### PR TITLE
request-history-filtering-by-risk-outcome & request-history-filtering-by-file-type steps

### PR DESCRIPTION
**Pull request by annachemic**

### CODE REVIEW

File: specs/features/request-history-filtering-by-file-type.feature [hunk 1]
```diff
@@ -5,10 +5,11 @@ Feature: request-history-filtering-by-file-type
         Given I am logged into the ui
 
     @TEST-163_164_165_180
-    Scenario Outline: I can filter the log by file type
+    Scenario Outline: I am able to filter the risks log
```
Comment (NadHodeGW) [inline specs/features/request-history-filtering-by-file-type.feature:8]: I updated this to "I can filter the log by file type", i'm not sure why it's changed back; please use the latest master to see latest changes & revert this back to previous state.

File: specs/features/request-history-filtering-by-file-type.feature [hunk 2]
```diff
@@ -5,10 +5,11 @@ Feature: request-history-filtering-by-file-type
...
         Given I have navigated to the Request History page
-        When I click on the Add Filter button and add a file type filter as <filter>
```
Comment (NadHodeGW) [inline specs/features/request-history-filtering-by-file-type.feature:10]: Same as above, please revert back to previous state

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
See requested changes in files

File: src/pages/request-history.page.js [hunk 3]
```diff
@@ -45,27 +45,27 @@ module.exports = {
...
-    fileTypeXlsx: "",
-    fileTypeXls: "",
-    fileTypeXlsm: "",
-    fileTypePpt: "",
-    fileTypePptx: "",
-    fileTypeJpeg: "",
-    fileTypePng: "",
-    fileTypeGif: "",
-    fileTypePdf: "",
+    fileTypeDoc: "div[class*='PopupFilter_inner__'] >  fieldset:nth-child(1) > div > label:nth-child(1) > span:nth-child(1) > span > input",
```
Comment (annachemic) [inline src/pages/request-history.page.js:48]: @NadHodeGW I know these locators look crazy but there is no ids or specific locators at all
Now it works with it using https://k8-proxy.github.io/request-history page

File: src/pages/request-history.page.js [hunk 4]
```diff
@@ -322,7 +332,7 @@ module.exports = {
     I.click(element);
   },
 
-  clickPrevious() {
+  clickNext() {
```
Comment (annachemic) [inline src/pages/request-history.page.js:335]: a typo, I assume

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**